src/test/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/adminofficer/AdminOfficerAppealOutcomePersonalisationTest.java

```diff
@@ -0,0 +1,113 @@
+    private String appellantGivenNames = "someAppellantGivenNames";
+    private String appellantFamilyName = "someAppellantFamilyName";
+
+    private String hearingCentre = "GLASGOW";
+
+    private String applicationDecision = "ALLOWED";
+
+    private String iaExUiFrontendUrl = "hhh";
+
+
+    @BeforeEach
+    public void setup() {
+
+        when(asylumCase.read(APPEAL_REFERENCE_NUMBER, String.class)).thenReturn(Optional.of(appealReferenceNumber));
+        when(asylumCase.read(ARIA_LISTING_REFERENCE, String.class)).thenReturn(Optional.of(ariaListingReference));
+        when(asylumCase.read(APPELLANT_GIVEN_NAMES, String.class)).thenReturn(Optional.of(appellantGivenNames));
+        when(asylumCase.read(APPELLANT_FAMILY_NAME, String.class)).thenReturn(Optional.of(appellantFamilyName));
+        when(asylumCase.read(HEARING_CENTRE, HearingCentre.class)).thenReturn(Optional.of(HearingCentre.GLASGOW));
+        when(asylumCase.read(IS_DECISION_ALLOWED, AppealDecision.class))
+                .thenReturn(Optional.of(AppealDecision.ALLOWED));
+
+        adminOfficerPersonalisationProvider = new AdminOfficerPersonalisationProvider(iaExUiFrontendUrl);
+
+        adminOfficerAppealOutcomePersonalisation = new AdminOfficerAppealOutcomePersonalisation(
+                decisionAndReasonUploadedTemplateId,
+                adminOfficerPersonalisationProvider,
+                emailAddressFinder
+        );
+    }
+
+    @Test
+    public void should_return_given_template_id() {
+        assertEquals(decisionAndReasonUploadedTemplateId, adminOfficerAppealOutcomePersonalisation.getTemplateId(asylumCase));
+    }
+
+    @Test
+    public void should_return_given_reference_id() {
+        assertEquals(caseId + "_APPEAL_OUTCOME_ADMIN",
+                adminOfficerAppealOutcomePersonalisation.getReferenceId(caseId));
+    }
+
+    @Test
+    public void should_throw_exception_on_personalisation_when_case_is_null() {
+
+        assertThatThrownBy(() -> adminOfficerAppealOutcomePersonalisation.getPersonalisation((AsylumCase) null))
+                .isExactlyInstanceOf(NullPointerException.class)
+                .hasMessage("asylumCase must not be null");
+    }
+
+    @Test
+    public void should_return_personalisation_when_all_information_given() {
+
+        Map<String, String> personalisation = adminOfficerPersonalisationProvider.getAdminPersonalisation(asylumCase);
+
+        assertEquals(appealReferenceNumber, personalisation.get("appealReferenceNumber"));
+        assertEquals(ariaListingReference, personalisation.get("ariaListingReference"));
+        assertEquals(appellantGivenNames, personalisation.get("appellantGivenNames"));
+        assertEquals(appellantFamilyName, personalisation.get("appellantFamilyName"));
+        assertEquals(hearingCentre, personalisation.get("hearingCentre"));
+        assertEquals(applicationDecision, personalisation.get("applicationDecision"));
+
+    }
+}
```

Comment:
Maybe add a test for when applicationDecision is refused to cover that scenario.
So you set the value to refused, and assert that it is so.

Reply:
Done.